### ISSUE TITLE
HeteroGCLSTM for edge regression

### ISSUE THREAD

**Issue by marcolomele** [benedekrozemberczki/pytorch_geometric_temporal#241]
Hi everyone, 

How can I pass edge features to a HeteroGCLSTM layer? Looking at the [documentation](https://pytorch-geometric-temporal.readthedocs.io/en/latest/modules/root.html#heterogeneous-graph-convolutional-layers) it seems that only node features can be passed.

I’m researching the use of graph neural networks for predicting economical interactions between countries. My goal is to predict edge weights (called labels in PyG's documentation), given nodes' features and edges' attributes. So far, I repreesnt temporal relations only through the input features. As such, my model considers monthly snapshots individually during the training and testing phase. 

Huge thanks to @benedekrozemberczki for making this library available to everyone!
